specs/sharding/beacon-chain.md

```diff
@@ -258,7 +299,10 @@ class ShardBlobBodySummary(Container):
     data_root: Root
     # Latest block root of the Beacon Chain, before shard_blob.slot
     beacon_block_root: Root
-    # TODO: fee payment amount fields (EIP 1559 like)
+    # fee payment fields (EIP 1559 like)
+    # TODO: express in MWei instead?
+    max_priority_fee_per_sample: Gwei
+    max_fee_per_sample: Gwei
```

Comment:
Wouldn't it be more clear to define?
- `priority_fee_per_sample`: kind of fixed priority bid 
- `max_base_fee_per_sample`: base fee is what a builder is uncertain about and thus setting an upper bound

Reply:
I'm following the EIP-1559 spec naming here, and the priority fee is dynamic, i.e. a proposer can choose to select a transaction that pays some of the priority fee towards the base fee instead, to be able to get it included.